latest_story.html

```diff
@@ -0,0 +1,103 @@
+<html>
+
+<head>
+    <title>html task</title>
+    <link rel="stylesheet" href="story.css" />
+   
+
+</head>
+
+<body>
+    <header>
+        <h1>Latest stories</h1>
+    </header>
+    <ul>
+
+
+        <li>
+            <article>
+                <img
+                    src="https://travel.home.sndimg.com/content/dam/images/travel/fullset/2015/10/12/new-seven-wonders-great-wall-of-china.jpg.rend.hgtvcom.616.462.suffix/1491581549051.jpeg" />
+                <p>December 11 2017 <br /></p>
+                <p class="a">Masood barkati, editor children's mag Naunehal, passes away </p>
+            </article>
+        </li>
+
+
+
+
+        <li>
+            <article>
+               
+ <img
+                    src="https://travel.home.sndimg.com/content/dam/images/travel/fullset/2015/10/12/new-seven-wonders-great-wall-of-china.jpg.rend.hgtvcom.616.462.suffix/1491581549051.jpeg" />
+                <p> Updated December 11 2017 <br /></p>
+                <p class="a">1,000 primary schools shut down in KP over low enrolment </p>
+            </article>
+        </li>
+
+
+
+
+
+        <li>
+            <article>
+                <img
+                    src="https://travel.home.sndimg.com/content/dam/images/travel/fullset/2015/10/12/new-seven-wonders-great-wall-of-china.jpg.rend.hgtvcom.616.462.suffix/1491581549051.jpeg" />
+
+
+                <p> Updated December 11 2017 <br /></p>
+                <p class="a">Kohli, Anushka set to marry in Italy </p>
+            </article>
+        </li>
+
+
+
+
+
+
+        <li>
+            <article>
+                <img
+                    src="https://travel.home.sndimg.com/content/dam/images/travel/fullset/2015/10/12/new-seven-wonders-great-wall-of-china.jpg.rend.hgtvcom.616.462.suffix/1491581549051.jpeg" />
+
+                <p> Updated December 11 2017 <br /></p>
+                <p class="a">Moeem urges England team to improve off-field behaviour </p>
+            </article>
+        </li>
+
+
+
+
+
+        <li>
+            <article>
+               
+                <img
+                    src="https://travel.home.sndimg.com/content/dam/images/travel/fullset/2015/10/12/new-seven-wonders-great-wall-of-china.jpg.rend.hgtvcom.616.462.suffix/1491581549051.jpeg" />
+            
+                <p> Updated December 11 2017 <br /></p>
+                <p class="a">A sour season for kinno exports</p>
+            </article>
+        </li>
+
+
+
+
+
+        <li class="bordernone">
+            <article>
+                <img
+                    src="https://travel.home.sndimg.com/content/dam/images/travel/fullset/2015/10/12/new-seven-wonders-great-wall-of-china.jpg.rend.hgtvcom.616.462.suffix/1491581549051.jpeg" />
+
+                <p> Updated December 11 2017<br /></p>
+                <p class="a">Enterpreneurs will be key player in corporate farming </p>
+            </article>
+        </li>
+
+
+
+    </ul>
+</body>
+
+</html>
```

Comment:
The code should look good to the eyes.  Remove unwanted empty lines and align the code.

Reply:
Removed unwanted empty lines.